src/main/java/homer/view/LaunchMeterView.java

```diff
@@ -69,23 +70,26 @@ public void start(final Stage stage) throws Exception {
 
         ElectricalMeterViewManager view = loader.getController();
         view.setMeter(meter);
-        final long millis = 10;
+        final long millis = 4;
         final long hours = 1;
-        for (int i = 0; i < outlets.size(); i++) {
-            Light light = lights.get(i);
-            Outlet outlet = light.getPowerInfo().getOutlet();
-            light.updateTick(Duration.ofMillis(millis));
-            outlet.setState(new OutletState().addValue(light.getInstantConsumption()));
-            outlet.updateTick(Duration.ofHours(hours));
-        }
-        meter.updateTick(Duration.ofHours(hours));
-        view.setLabels();
-
-        // Create a scene and show the stage
         scene = new Scene(root);
         stage.setTitle(TITLE);
         stage.setScene(scene);
         stage.show();
+        Timeline timeline = new Timeline(new KeyFrame(javafx.util.Duration.millis(300), event -> {
+            for (int i = 0; i < lights.size(); i++) {
+                Light light = lights.get(i);
+                Outlet outlet = light.getPowerInfo().getOutlet();
+                light.updateTick(Duration.ofMillis(millis));
+                outlet.getState().addValue(light.getInstantConsumption());
+                outlet.updateTick(Duration.ofHours(hours));
+            }
+            meter.updateTick(Duration.ofHours(hours));
+            view.setLabels();
+        }));
+        timeline.setCycleCount(Timeline.INDEFINITE);
+        timeline.play();
+
```

Comment:
Updates the view every 300ms (about 3 fps) indefinitely.
In particular the view is updated by calling `view.setLabels()`.
This is just a showcase for test purposes, the real implementation of this code is going to be in the main loop.

Looks like it's working really well, better then I expected actually.

Though I don't really like how `Outlet`'s state must be updated before calling `updateTIck()`, maybe I should instantiate Outlets with a default state value, which should be a little over 0.0 (e.g. 0.0001 should be enough), but I'd rather not hardcode it. I'll go through it and hope there's no need to heavily refactor `Outlets`/`PoweredDevice`/`PoweredDeviceInfo`. 

Using a `Timeline` object might be useful for updating the view indipendently from the rest of the loop, it updates the view at a fixed frequency, and if run on another thread shouldn't be slowed down by the code (Shouldn't become unresponsive during long calcs, and would be updated piece by piece).

@smagnaniuni @michelececcacci what do you think?

Reply:
Should be good, was thinking of using a timeline for the graphs too but have no idea how to use it at the moment.
Have to review later